src/bg_mpl_stylesheets/styles.py

```diff
@@ -102,7 +102,7 @@ def update_style_with_latex(style):
     return style
 
 
-all_styles = {"bg_style": bg_style}
+all_styles = {"bg-style": bg_style}
```

Comment:
This only place that remains as `bg_style` is the variable name since we are not able to use `-`.

Reply:
yes, this is correct.